src/components/LabListEditor/LabListEditor.tsx

```diff
@@ -27,12 +29,24 @@ export function LabListEditor({labs, prefix, moduleId}: Props) {
           <Row style={{marginBottom: '1rem'}}>
             <Col>Labs</Col>
             <Col className='d-flex justify-content-end align-items-center'>
-              <IconButton
-                icon={faPlusCircle}
-                size={'2x'}
-                link={getNewLabEditorLink(moduleId)}
-                color={'black'}
-              />
+              { disabledModule ?
+                <>
+                  <ReactTooltip place='left' type='dark' effect='solid'/>
+                  <a data-tip='"Disable Module" must be unchecked to add a Lab'>
+                    <IconButton
+                      icon={faPlusCircle}
+                      size={'2x'}
+                      color={'grey'}
+                    />
+                  </a>
```

Comment:
Instead of wrapping the IconButton in an achor tag, is it  possible to pass the tooltip text to IconButton and have it handle the tooltip?

Reply:
I'll chime in here since I am also using a Tooltip for the search bar, react-bootstrap has a Tooltip component, which I'm using for the search bar (#69), I created it as a separate component, which can be made to do more if needed. You should be able to wrap anything around the ToolTip, a button, link, etc, and it will show. Ex: `<ToolTip ..props..><IconButton /></ToolTip>`. I think this is the "std" way of doing it, what do you suggest?